public/app/features/panel/panel_ctrl.ts

```diff
@@ -154,6 +162,16 @@ export class PanelCtrl {
       });
     }
 
+    // TS hack :<
+    if ('datasource' in this && this['datasource'].supportsExplore) {
```

Comment:
Humpf Maybe throw it into a function for now? Not a blocker but would be nice to keep the none feature flags parts of the codebase as clean as possible.